### PR TITLE
Add archive artefact stage to linux new installers

### PR DESCRIPTION
**Pull request by gdams**
This PR archives the generated deb/rpm files in jenkins.

### CODE REVIEW

File: linuxNew/Jenkinsfile [hunk 1]
```diff
@@ -50,7 +51,7 @@ def buildAndTest() {
             break;
     }
     for (os in ["Debian", "RedHat", "Suse"]) {
-        sh("./gradlew clean packageJdk${os} checkJdk${os} --parallel -PPRODUCT=${PRODUCT} -PPRODUCT_VERSION=${VERSION}")
+        sh("./gradlew packageJdk${os} checkJdk${os} --parallel -PPRODUCT=${PRODUCT} -PPRODUCT_VERSION=${VERSION}")
```
Comment (sophia-guo) [inline linuxNew/Jenkinsfile:54]: It's been updated. So we don't need the for loop any more. 

`./gradlew --parallel check -PPRODUCT=${PRODUCT} -PPRODUCT_VERSION=${VERSION} --stacktrace` should be good?

**Comment by sophia-guo** [comment]
Will also add check-ca-certificates job in a separate PR.

File: linuxNew/Jenkinsfile (hunk 1 above)
Comment (gdams) [reply linuxNew/Jenkinsfile]: In the most recent refactor I've broken the jobs down into sub-jobs that in parallel for each version/distro.

Comment (sophia-guo) [reply linuxNew/Jenkinsfile]: A second thought, with the parallel for each version is it possible or easy to only generate one version of installers with jenkinsfile?

**Pull request merged**